tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -279,3 +279,38 @@ function testInvokingFunctionContainingIncludedRecordParamOfTypeOtherThanRecordO
     _ = obj.fn(f = {});
     _ = obj.fn2(f = {});
 }
+
+type Record5 record {
+};
+
+type Record6 record {
+};
+
+function fn5(*Record5 record5) {
+}
+
+function fn6(*Record5 record5, *Record6 record6) {
+}
+
+function testInvokingFunctionWithNamedArgForIncludedRecordParam() {
+    fn5(record5 = {}, j = 2);
+    fn5(j = 2, record5 = {});
+    fn6(record5 = {}, j = 2);
+    fn6(record5 = {}, record6 = {}, j = 2);
+    fn6(j = 2, record5 = {}, record6 = {});
+}
+
+type Baz record {|
+    int r;
+|};
+
+function fn7(*Baz r) {
+}
+
+function fn8(string k, *Record5 record5) {
+}
+
+function testInvokingFunctionWithPositionArgForIncludedRecordParam() {
+    fn8("abc", {}, j = 2);
+    fn8("abc", {}, j = 2, i = 10);
```

Comment:
Can we add a few more tests similar to the named ones (with multiple included record params)?

Reply:
We can reuse the same functions.

Reply:
Added test cases